build.sbt

```diff
@@ -16,23 +18,21 @@ lazy val `scala-exercises` = (project in file("."))
 
 // Purely functional core
 
-lazy val core = (crossProject in file("core"))
+lazy val core = (crossProject(JSPlatform, JVMPlatform) in file("core"))
```

Comment:
I think we just need JVMPlatform, aren't we?

Reply:
The core project needs cross compilation because the client depends on the js core and the server needs the jvm core.